internal-docs/RELEASE.md

```diff
@@ -0,0 +1,10 @@
+# RECCO PLUGIN: PUBLISHING A NEW RELEASE
+
+## Create a pull request
+Open a PR with the following changes:
+- Update the Recco version in the top level `pubspec.yaml` file to the new version, following Semantic Versioning (MAJOR.MINOR.PATCH).
+- In the readme file, update any references mentioning the current version.
+- Add a new entry to the changelog file describing all the changes contained in this new release.
```

Comment:
it would be nice to put the links to the README.md and CHANGLOG.md files

Reply:
thanks Miguel for pointing that out, already did 👍